src/platform/site-wide/cta-widget/helpers.js

```diff
@@ -111,33 +81,32 @@ export const toolUrl = appId => {
   switch (appId) {
     case widgetTypes.HEALTH_RECORDS:
       return {
-        url: mhvUrl('download-my-data'),
+        url: mhvUrl(hasSessionSSO(), 'download-my-data'),
```

Comment:
I don't think this is going to work out the way you expect it to.  The `hasSessionSSO` local storage variable can have two possible values, `"true"` or `"false"`.  The former means they have an active SSOe session and the later means they don't.  If no value is stored, that means we don't know.  Thus I'm thinking you meant something like ...

```
  url: mhvUrl(hasSessionSSO() === 'true', 'download-my-data'),
```

Reply:
Going to change this so index.js passes in `useSSOe` as a parameter.